src/View/Rgpd/index.html.twig

```diff
@@ -0,0 +1,38 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}RGPD
+{% endblock %}
+
+{% block stylesheet %}
+	{{parent()}}
+	<link rel="stylesheet" href="/assets/css/rgpd.css">
+{% endblock %}
+
+{% block content %}
+	<main>
+		<div class="rgpd-container">
```

Comment:
does this div tag has any meaning ?

Reply:
You mean the div class name ? Yes, it exists so I can use custom styles on the <p> and the <h1> without giving them classes.